src/admin/js/ets-admin.js

```diff
@@ -0,0 +1,37 @@
+jQuery(document).ready(function ($) {
```

Comment:
@ravisoni6262 we want to distance ourselves from jQuery, can we refactor it using Vanilla JS?